### PR TITLE
Move ERP Guides from CMS and Control Panels to Enterprise Resource Planning section

### PR DESCRIPTION
**Pull request by bagasme**
Currently, all ERP guides (mainly Odoo installation) resides on `CMS and Control Panels` section of `Websites` category. This PR separate them into `Enterprise Resource Planning Applications` section instead.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit c48f4f370524198f37fe9aacaad184d75baae232

https://deploy-preview-2801--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by bagasme** [comment]
So I got CI error that indicates `404 error` when accessing `/docs` URL.

However, on my local build the mentioned URL can be accessed as usual (and CI should throw error on `test_uppercase_alias` and `test_underscore_alias`) AFAIK.

Perhaps any suggestion?

**Comment by Rajakavitha1** [comment]
Hi @bagasme  Thanks for the PR!
We have a front matter in Hugo called aliases that helps with this.
It looks like you need to set an alias from the old guide path to the new guide path inside the guide’s front matter.

**Comment by bagasme** [comment]
@Rajakavitha1 But I had (correctly) set up `aliases` as you mentioned.

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 But I had (correctly) set up `aliases` as you mentioned.

Yes, I saw that  in https://github.com/linode/docs/tree/c48f4f370524198f37fe9aacaad184d75baae232 You will have to set it in every guide that you moved as well.

**Comment by bagasme** [comment]
So how can I set up `aliases` on each moved guides, given that I had also set up on moved folder (in this case, `erp`)?

**Comment by Rajakavitha1** [comment]
> So how can I set up `aliases` on each moved guides, given that I had also set up on moved folder (in this case, `erp`)?
For example in ```install-an-odoo-11-stack-on-ubuntu-16-04``` guide that you moved add ```aliases: ['websites/cms/install-an-odoo-11-stack-on-ubuntu-16-04']`` in the front matter of the guide.

**Comment by nmelehan** [comment]
Hi @bagasme--I'll push up a commit in a few minutes that demonstrates what @Rajakavitha1 is mentioning, so hopefully it'll be a bit clearer

File: docs/websites/erp/install-an-odoo-11-stack-on-ubuntu-16-04/index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ title: 'Install an Odoo 11 Stack on Ubuntu 16.04'
 contributor:
   name: Damaso Sanoja
   link: https://twitter.com/damasosanoja
+aliases: ['websites/cms/install-an-odoo-11-stack-on-ubuntu-16-04/']
```
Comment (bagasme) [inline docs/websites/erp/install-an-odoo-11-stack-on-ubuntu-16-04/index.md]: Oh dear, that are the alias to old URL.

**Comment by bagasme** [comment]
@nmelehan so next time when refactoring/moving guides should I use `mv_guide.py` script that automatically set up aliases for me?

**Comment by bagasme** [comment]
Travis build passed but here still `pending`...

**Comment by bagasme** [comment]
So can we start CI test?

**Comment by Rajakavitha1** [comment]
> So can we start CI test?

Yes, I guess we should start a CI test. @nmelehan  could you please help us with this.

**Comment by Rajakavitha1** [comment]
Hi @bagasme !!!!
Linode appreciates your contributions in fixing the doc issues.
Could you please drop an email to aguarisma@linode.com to see if we can reward you for your efforts.

**Comment by nmelehan** [comment]
hi @bagasme --

If you click Details link on the Travis check, it'll show that all the test passed. I'm not sure why it's still showing as Pending in this pull request, but I think it was probably just a temporary problem with the webhook that Travis uses to notify GitHub. In any case, we will still be able to merge this PR, but I still need to have our manager review it. I'll add a comment here when we have finished that review. Thank you!

File: docs/websites/erp/install-an-odoo-11-stack-on-ubuntu-16-04/index.md (hunk 1 above)
Comment (nmelehan) [reply docs/websites/erp/install-an-odoo-11-stack-on-ubuntu-16-04/index.md]: we set the old URL in the frontmatter under the `aliases` property, and Hugo uses that information to build a redirect from that URL to the new location

**Pull request merged**